DuckDuckGo/TabViewController.swift

```diff
@@ -2271,7 +2271,7 @@ extension TabViewController: SecureVaultManagerDelegate {
         SecureVaultErrorReporter.shared.secureVaultInitFailed(error)
     }
     
-    func secureVaultManagerIsEnabledStatus(_: SecureVaultManager) -> Bool {
+    func secureVaultManagerIsEnabledStatus(_ manager: SecureVaultManager, forType type: AutofillType?) -> Bool {
```

Comment:
It looks  like you need to do the same in EmailSignupViewController

Reply:
😞 I missed a push-  Sorry about that.